File: app/src/main/java/com/example/csce3615_moblieapp_project/Main_Campus.java
package com.example.csce3615_moblieapp_project;

import android.content.Context;
import android.content.res.Resources;
import android.graphics.drawable.Drawable;
import android.graphics.Canvas;
import android.media.Image;
import android.os.Bundle;
import android.view.SurfaceView;
import android.view.ViewGroup;
import android.widget.ImageView;

import androidx.core.content.res.ResourcesCompat;

import com.example.csce3615_moblieapp_project.R;


public class Main_Campus extends SurfaceView {
    private Context context;
    Resources res = context.getResources();
    public Main_Campus(Context context) {
        super(context);
    }
    @Override
    public void draw(Canvas canvas) {
        super.draw(canvas);
        Drawable myImage = ResourcesCompat.getDrawable(res, R.drawable.main_campus, null);
    }
}



    /*

    Drawable drawable  = Drawable.createFromPath(image);
    //image.setImageDrawable(drawable);

        public void setDrawable(Drawable drawable) {
            this.drawable = drawable;
        }

        public void setImage(String image) {
            this.image = image;
        }
 */

File: app/src/main/java/com/example/csce3615_moblieapp_project/MainCampus.kt
package com.example.csce3615_moblieapp_project

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.tooling.preview.Preview
import com.example.csce3615_moblieapp_project.ui.theme.CSCE3615_MoblieApp_ProjectTheme

class MainCampus : ComponentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            CSCE3615_MoblieApp_ProjectTheme {
                // A surface container using the 'background' color from the theme
                Surface(
                    modifier = Modifier.fillMaxSize(),
                    color = MaterialTheme.colorScheme.background
                ) {
                    appMainCampus()
                }
            }
        }
    }

    @Preview(showBackground = true)
    @Composable
    fun appMainCampus() {
        CSCE3615_MoblieApp_ProjectTheme {
            Image(painter = painterResource(id = R.drawable.main_campus), contentDescription = null, contentScale = ContentScale.Crop, alpha = 1f)
        }
    }

}